workshops/erc20-mts/Makefile

```
@@ -0,0 +1,25 @@
+	./scripts/mint-erc20.sh
+
+balance1-cast:
+	./scripts/erc20-balance-user1.sh
+
+balance1-cli:
+	./scripts/cli-balances-user1.sh
+
+deploy-wasm:
+	./scripts/deploy-wasm.sh
+
+.PHONY: all add-keys deploy-erc20 balance1-cast balance1-cli deploy-wasm
```

checkmake (0.2.2)

[warning] Missing required phony target "clean"

(minphony)